titanic.ipynb

```diff
@@ -43,9 +43,6 @@
         "# Load the Titanic dataset\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 8 to 9_
```python
# Convert the 'age' column from float to integer using pandas
titanic['age'] = titanic['age'].apply(pd.to_numeric, downcast='integer', errors='coerce')
```
Test

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/26)._